recipes/xplique/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "xplique" %}
+{% set version = "0.3.1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/Xplique-{{ version }}.tar.gz
+  sha256: 1a7eb7ff6fbd88996e25b4398b478008d1e4e7ad725346d6cbce3c39b09c613a
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - python >=3.6
+    - matplotlib-base
+    - numpy
+    - opencv
+    - scikit-image
+    - scikit-learn
+    - scipy
+    - tensorflow >=2.1.0
+
+test:
+  imports:
+    - tests
+    - tests.attributions
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://pypi.org/project/Xplique/
+  summary: Explanations toolbox for Tensorflow 2
+  license: MIT
+  license_file: LICENSE
+  description: |
+    🦊 <b>Xplique</b> (pronounced <i>\ɛks.plik\</i>) is a Python toolkit dedicated to explainability, currently based on Tensorflow.
+    The goal of this library is to gather the state of the art of Explainable AI to help you understand your complex neural network models.
+
+    PyPI: [https://pypi.org/project/Xplique/](https://pypi.org/project/Xplique/)
+
```

Comment:
I'm skeptical unicode is a good idea here. Can we only use ascii please?

Reply:
Ok. I will update it accordingly.